war/src/main/scss/modules/buttons.scss

```diff
@@ -1,9 +1,11 @@
+@use "../abstracts/mixins";
+
 .jenkins-button {
   --item-background--hover: var(--button-background--hover);
   --item-background--active: var(--button-background--active);
   --item-box-shadow--focus: var(--button-box-shadow--focus);
 
-  .item();
+  @include mixins.item;
 
   appearance: none;
   display: inline-flex;
```

Comment:
It appears for me?

<img width="449" alt="image" src="https://user-images.githubusercontent.com/43062514/233383693-0869c19a-d237-46e9-b27d-07a72ffc8a22.png">

Reply:
The fact that it appears in the compiled CSS is the potential problem I was pointing out, since it does not appear in the compiled CSS before this PR.

Reply:
To clarify, here is the diff of `.jenkins-button` for the old compiled CSS and the new compiled CSS after this PR:

```
@@ -909,6 +867,7 @@
   --item-box-shadow--focus: var(--button-box-shadow--focus);
   position: relative;
   z-index: 0;
+  border-radius: 0.66rem;
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
```

Here is the new CSS:

```
.jenkins-button {
  --item-background--hover: var(--button-background--hover);
  --item-background--active: var(--button-background--active);
  --item-box-shadow--focus: var(--button-box-shadow--focus);
  position: relative;
  z-index: 0;
  border-radius: 0.66rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: inline-flex;
  align-items: center;
  justify-content: center;
  border: none;
  outline: none;
  margin: 0;
  padding: 0.5rem 0.85rem;
  font-size: 0.8rem;
  font-weight: 500;
  text-decoration: none !important;
  background: transparent;
  color: var(--text-color) !important;
  border-radius: 0.66rem;
  cursor: pointer;
  min-height: 36px;
  white-space: nowrap;
  gap: 1ch;
}
```

So this is not so much a bug per se but rather the introduction of a duplicate `border-radius` where there was no duplicate before.

Reply:
Ah seems like SCSS isn't removing duplicates like LESS was in that case. Should be okay now.